content/updates/2018-jan.md

```diff
@@ -0,0 +1,21 @@
+---
+title: January 2018
+date: 2018-02-07
+hidden: true
+image: image  # pending pi-image
+update: update  # pending pi-image
+---
+
+- Support for [custom Arduino commands](/api/servo-assembly/arduino-commands/)
+- Implement [Cartesian coordinate system](/api/vision/coordinates/#cartesian-coordinates)
+- Fix issue with camera after code terminates
+- Remove dates from logs (They were wrong anyway)
+- Remove unnecessary noisy log messages
+- Remove unnecessary `size` attribute from a marker
+- Deprecate broken `polar` coordinate system in favour of [`spherical`](/api/vision/coordinates/#spherical-coordinates)
+- Flash the start LED once robot is ready (Also reduced chance of press not being registered)
+- Make camera return items when indexing rather than interacting
```

Comment:
I'm not sure what this means.

Given the code changes which happened, I think you're referring to the change to `ResultList` so that doing `markers[0]` actually worked? If so, maybe we should word this as "Make indexing into a list of markers actually return a marker instance, instead of `None`"?